### ISSUE TITLE
Error with directories

### ISSUE THREAD

**Issue by dazinator** [tillig/plantuml-in-azdo-wiki#2]
Thanks for this, it looks really good.
All was working in VSCode for me, until I moved my `.puml files into a sub directory (named v2.0.0), and then renamed the subfolder. I saw an error like this:

```
> plantuml-in-azdo-wiki@1.0.0 watch C:\Users\**\source\repos\**-doc-architecture-roadmap
> node index.js

[13:05:45] Generating C:\Users\**\source\repos\**-doc-architecture-roadmap\diagrams\1.2 Overview .png
[13:05:45] Generating C:\Users\**\source\repos\**-doc-architecture-roadmap\diagrams\1.2 Overview Alpha.2.png
[13:05:45] Generating C:\Users\**\source\repos\**-doc-architecture-roadmap\diagrams\1.2.png
[13:05:45] Generating C:\Users\**\source\repos\**-doc-architecture-roadmap\diagrams\v2.png
[13:05:45] Generating C:\Users\**\source\repos\**-doc-architecture-roadmap\diagrams\v1.1.png
[13:06:45] Generating C:\Users\**\source\repos\**-doc-architecture-roadmap\docs\v2.0.0.png
events.js:174
      throw er; // Unhandled 'error' event
      ^

Error: EISDIR: illegal operation on a directory, read
Emitted 'error' event at:
    at lazyFs.read (internal/fs/streams.js:165:12)
    at FSReqWrap.wrapper [as oncomplete] (fs.js:462:17)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! plantuml-in-azdo-wiki@1.0.0 watch: `node index.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the plantuml-in-azdo-wiki@1.0.0 watch script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\**\AppData\Roaming\npm-cache\_logs\2020-10-27T13_06_45_858Z-debug.log
The terminal process "C:\WINDOWS\System32\WindowsPowerShell\v1.0\powershell.exe -Command npm run watch" terminated with exit code: 1.

```

**Comment by tillig** [comment]
In an earlier version of the comment it sounded like you'd also found a fix. Is that something you could submit as a pull request?

**Comment by tillig** [comment]
Resolved by #3.
